### DISCUSSION TITLE
I just want to report a typo.

### DISCUSSION THREAD

**Discussion by arm75** [TanStack/table#4608] [General]
on the page: https://react-table-v7.tanstack.com/docs/quick-start , about 30% down the page, under the section "Using the useTable hook", the text "at the very least needs to be provided with an object containing the memoized", should be "memorized". thank you that is all.

**Comment by lukethacoder** [comment]
Pretty sure **memoized** is intentional. See the [React Docs](https://reactjs.org/docs/hooks-reference.html#usememo)

**Comment by KevinVandy** [comment]
Memoized is correct and different from memorized. It's where the react useMemo hook gets it's name
